libs/plugins/stream-client/src/lib/core/models/generate-state-from-resource.ts

```diff
@@ -17,6 +17,7 @@ export function generateStateFromResource(resource: any, schemas: ContributionSc
     triggers: originalTriggers,
     currentSelection,
     triggerConfigure,
+    stageConfigure,
```

Comment:
It seems that `resource` won't have `triggerConfigure`, `currentSelection` and `stageConfigure`. I realize that we have them in object destructuring as we need to satisfy the type check of `streamActions.Init` class's payload. 

If we had implemented the interface of `resource` it would have shown us the error.